airlock/templates/file_browser/request/filegroup.html

```diff
@@ -15,25 +15,21 @@
 </style>
 {% #card title=group.title container=True class="group_modal" %}
   <div data-testid="c3">
-    <form action="{{ group.c2_edit_url }}" method="POST" aria-label="group-edit-form">
+    <form
+      action="{{ group.c2_edit_url }}"
+      aria-label="group-edit-form"
+      class="grid grid-cols-2 gap-x-6 gap-y-4 items-start"
+      method="POST"
+    >
       {% csrf_token %}
-      <div style="display: flex; flex-direction: row; width: 100%; flex-wrap: wrap; gap: 1rem;">
-        <div style="display: flex: flex-direction: column; flex-basis: 100%; flex: 1;">
-
-          {% form_textarea field=group.c2_edit_form.context resize=True hint_below=True show_placeholder=True placeholder="Describe the data to be released in this group of files" class="w-full max-w-lg mx-auto" rows=6 disabled=group.c2_readonly readonly=group.c2_readonly %}
-        </div>
-        <div style="display: flex: flex-direction: column; flex-basis: 100%; flex: 1; gap: 1rem;">
-          {% form_textarea field=group.c2_edit_form.controls resize=True hint_below=True show_placeholder=True placeholder="Describe the disclosure controls that have been applied to these files" class="w-full max-w-lg mx-auto" rows=6 disabled=group.c2_readonly readonly=group.c2_readonly %}
-        </div>
-      </div>
+      {% form_textarea field=group.c2_edit_form.context show_placeholder=True resize=True placeholder="Describe the data to be released in this group of files" class="w-full max-w-lg mx-auto" rows=6 disabled=group.c2_readonly readonly=group.c2_readonly %}
```

Comment:
Is there a reason to remove the `hint_below=True` on these two fields? I deliberately put it below the field because otherwise the two text boxes aren't aligned.

Reply:
Nope, this was rebase fail, good spot I'll fix now!